packages/types/app/vue.d.ts

```diff
@@ -11,6 +11,8 @@ declare module 'vue/types/options' {
   interface ComponentOptions<V extends Vue> {
     asyncData?(ctx: Context): Promise<object | void> | object | void
     fetch?(ctx: Context): Promise<void> | void
+    fetchOnServer?: boolean
+    fetchDelay?: number
```

Comment:
Assuming https://github.com/nuxt/nuxt.js/blob/dev/packages/vue-app/template/mixins/fetch.server.js#L35 & alphabetical order :

```suggestion
    fetchDelay?: number
    fetchOnServer?: boolean | (() => boolean)
```

Reply:
Oh yes, I based my PR on the signature in the doc. It didn't even occur to me to check the logical order. Thanks for the review and correction! :)